tests/lib/tools/suite/tests.cleanup/task.yaml

```diff
@@ -0,0 +1,74 @@
+summary: tests for the tests.cleanup tool
+systems: [ubuntu-16.04-64]
+prepare: |
+    apt-get install -y shellcheck
+restore: |
+    apt-get remove --purge -y shellcheck
+execute: |
+    # The script passes shellcheck.
+    shellcheck "$TESTSTOOLS"/tests.cleanup
```

Comment:
I've recently realized that our spread shellcheck uses shellcheck from a snap (which ships a relatively new version of shellcheck). Older ubuntus (in particular 16.04) use ancient versions of shellcheck, which fail on things that are fine for new versions. I'd recommended to use a version from snap as well here to avoid annoying surprises in the future.

Reply:
I would have done that but this suite runs without snapd being present. I will think about how to get the best of both worlds somehow.

Reply:
Ah, I see. Can you use 20.04 for this test then? It would have shellcheck that is closer to what we use normally with version from the snap.

Reply:
nvm, you would have snapd then with 20.04 ;)

Reply:
We could use pre-built shellcheck binary from https://github.com/koalaman/shellcheck - it is available for x86 and arm. We could install it in the suite prepare so that all the tests here can benefit.